xtask/src/opt.rs

```diff
@@ -84,6 +85,14 @@ pub struct DocOpt {
     pub warning: WarningOpt,
 }
 
+/// Generate Rust code for device path nodes.
```

Comment:
Considering that the action is called simply `gen-code`, and that we might have some more automatically generated code in the future, maybe it's a better idea to make this doc comment more generic?

Reply:
Updated to be more general, and added a second line that explains where to look for more info (that second line will show up with `cargo xtask gen-code --help`.